### DISCUSSION TITLE
A real exploit unlike the "new exploit found" discussion that enables ChromeOS developer mode

### DISCUSSION THREAD

**Discussion by HelloIloveGimkit2** [3kh0/ext-remover#1287] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
[admvp (Activating Developer Mode Via Powerwash)](https://admvp.vercel.app)
I did steal the sh1mmer website source code but idrc

**Comment by py660** [comment]
unfortunately, this was already discovered

**Reply by HelloIloveGimkit2** [reply]
bro I was the one who fucking discovered it so don't even fucking try to say that this was already discovered

**Reply by HelloIloveGimkit2** [reply]
if it was before I even found it while doing br1ck I didn't skid from them so don't fucking try to lower the status of this exploit

**Reply by HelloIloveGimkit2** [reply]
someone else found it because I didn't say that I had found it the only reason your saying this is cuz I never said that I had it happen to me before that dude did br1ck

**Reply by awsome-webdev** [reply]
what is it with you and starting comment wars

**Reply by akabutnicer** [reply]
alright we get the point gimkit guy

**Reply by jeffrey6576i** [reply]
im crying

**Reply by py660** [reply]
Quote from @HelloIloveGimkit2 
> bro I was the one who fucking discovered it so don't even fucking try to say that this was already discovered

There is absolutely no reason to use profanity here. Please keep this discussion civil.

It is completely possible for you to ave discovered a workaround independently from others, while still having the same idea. Many isolated and distant ancient civilizations all came up with the idea of farming independently from each other in the agricultural revolution. Although you might have originally thought that your method would get merged into the repo, unfortunately not all suggestions get accepted.

**Reply by jeffrey6576i** [reply]
valid

**Reply by akabutnicer** [reply]
> Quote from @HelloIloveGimkit2
> 
> > bro I was the one who fucking discovered it so don't even fucking try to say that this was already discovered
> 
> There is absolutely no reason to use profanity here. Please keep this discussion civil.
> 
> It is completely possible for you to ave discovered a workaround independently from others, while still having the same idea. Many isolated and distant ancient civilizations all came up with the idea of farming independently from each other in the agricultural revolution. Although you might have originally thought that your method would get merged into the repo, unfortunately not all suggestions get accepted.

exactly.

**Reply by HelloIloveGimkit2** [reply]
> what is it with you and starting comment wars

cuz people are fucking dumbasses

**Reply by HelloIloveGimkit2** [reply]
> Quote from @HelloIloveGimkit2
> 
> > bro I was the one who fucking discovered it so don't even fucking try to say that this was already discovered
> 
> There is absolutely no reason to use profanity here. Please keep this discussion civil.
> 
> It is completely possible for you to ave discovered a workaround independently from others, while still having the same idea. Many isolated and distant ancient civilizations all came up with the idea of farming independently from each other in the agricultural revolution. Although you might have originally thought that your method would get merged into the repo, unfortunately not all suggestions get accepted.

ok I'm sorry for that but I did find it sorry for tryna start smth

**Reply by awsome-webdev** [reply]
> > what is it with you and starting comment wars
> 
> cuz people are fucking dumbasses

maybe your just to aggressive

**Comment by jeffrey6576i** [comment]
dont work.

**Reply by HelloIloveGimkit2** [reply]
keep trying

**Reply by jeffrey6576i** [reply]
i cant risk it again

**Comment by ghost** [comment]
does it still work if I don't have the cancel option on the confirm returning to secure mode part?

**Reply by jeffrey6576i** [reply]
no

**Comment by CHROMEOSKILLA** [comment]
holy fuck this is what I was looking for

**Reply by akabutnicer** [reply]
no need for profanity

**Reply by Luigi-Master** [reply]
lol

**Comment by CHROMEOSKILLA** [comment]
does this work on chrome 120

**Reply by akabutnicer** [reply]
it works on pretty much v100+ (at the moment, where v131 is the max)

**Reply by Luigi-Master** [reply]
bro why are you writing anyway on a month year old discussion

**Reply by akabutnicer** [reply]
plus im sure there are versions before v100 that work as well

**Reply by akabutnicer** [reply]
> bro why are you writing anyway on a month year old discussion

eh at least we all are writing on a month old discussion

**Reply by akabutnicer** [reply]
also 'month year old' is crazy

**Reply by Luigi-Master** [reply]
no I was talking about chromeoskilla

**Reply by akabutnicer** [reply]
both of my messages still stand perfectly fine

**Reply by Luigi-Master** [reply]
I meant when he had to bring this discussion back up by chatting in it, like admvp was taken off the repo

**Reply by schoolexploitkid** [reply]
it didnt work for me
